oolab/src/main/java/agh/ics/oop/AbstractWorldMap.java

```diff
@@ -20,13 +20,14 @@ public boolean isOccupied(Vector2d position) {
     public boolean place(Animal animal) {

```

Comment:
skoro rzucamy wyjątkiem to nie zwrócimy tu nigdy false -> ta metoda nie powinna już zwracać boolean